coderd/database/queries/users.sql

```diff
@@ -250,3 +250,14 @@ SET
 WHERE
 	id = $1
 RETURNING *;
+
+
+-- name: UpdateInactiveUsersToDormant :many
+UPDATE
+    users
+SET
+    user_status = 'dormant'::user_status
+WHERE
+    last_seen_at < @last_seen_after :: timestamp
```

Comment:
note for readers: this is correct as `last_seen_at` is `timestamp without time zone`.